neo/Persistence/LevelDB/Store.cs

```diff
@@ -0,0 +1,69 @@
+using Neo.IO.Data.LevelDB;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Reflection;
+
+namespace Neo.Persistence.LevelDB
+{
+    public class Store : IStore
+    {
+        private const byte SYS_Version = 0xf0;
+        private readonly DB db;
+
+        public Store(string path)
+        {
+            this.db = DB.Open(path, new Options { CreateIfMissing = true });
+            if (db.TryGet(ReadOptions.Default, SliceBuilder.Begin(SYS_Version), out Slice value) && Version.TryParse(value.ToString(), out Version version) && version >= Version.Parse("2.9.1"))
+                return;
+            WriteBatch batch = new WriteBatch();
```

Comment:
It's not faster removing the whole folder? (could be done in other PR)

Reply:
I don't want to change leveldb things too much because they will be removed soon...